packages/flutter_tools/flutter_analysis_options

```diff
@@ -1,8 +1,10 @@
 # Specify analysis options.
 #
-# Note that until there is a default "all-in" lint rule-set we need
-# to opt-in to all desired lints (https://github.com/dart-lang/sdk/issues/25843).
+# Note that until there are meta linter rules, we need to opt-in
```

Comment:
super nit: let's remove the "Note that" part, since it's just words that add nothing.